src/common/copilot/PowerPagesCopilot.ts

```diff
@@ -39,18 +38,16 @@ export class PowerPagesCopilot implements vscode.WebviewViewProvider {
   private loginButtonRendered = false;
   private telemetry: ITelemetry;
 
-  constructor(private readonly _extensionUri: vscode.Uri, _context: vscode.ExtensionContext, telemetry: ITelemetry, cliPath: string) {
+  constructor(private readonly _extensionUri: vscode.Uri, _context: vscode.ExtensionContext, telemetry: ITelemetry, pacWrapper: PacWrapper) {
     this.telemetry = telemetry;
     this._extensionContext = _context;
-    const pacContext = new PacWrapperContext(_context, telemetry);
-    const interop = new PacInterop(pacContext, cliPath);
-    this._pacWrapper = new PacWrapper(pacContext, interop); //For Web Terminal will not be available
+    this._pacWrapper = pacWrapper;
 
     _context.subscriptions.push(
```

Comment:
Likewise, for any disposables here, you can just add them to this class's `_disposables` instead of the extension's, as they'll be handled by the disposal of the class instance